modules/mirror-registry-flags.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-flags_{context}"]
+= Mirror registry for {product-title} flags
+
+The following flags are available with local host and remote mirror registries:
+
+[options="header",cols="1,3"]
+|===
+| Flags | Description
+| `--autoApprove` | A boolean value that disables interactive prompts. If set to `true`, the `quayRoot` directory is automatically deleted when uninstalling the mirror registry. Defaults to `false` if left unspecified.
+| `--initPassword` | The password of the init user created during Quay installation. Must be at least eight characters and contain no whitespace.
+| `--quayHostname` | The value to set `SERVER_HOSTNAME` in the Quay `config.yaml`. Defaults to `<targetHostname>:8443` if left unspecified. ^[1]^
+| `--quayRoot`, `-r` | The directory where Quay persistent data is saved, including `rootCA.key`, `rootCA.pem`, and `rootCA.srl` certificates. Requires about 12 GB for {product-title} 4.10 Release images, or about 358 GB for {product-title} 4.10 Release images and {product-title} 4.10 Operator images. Defaults to `/etc/quay-install` if left unspecified.
+| `--ssh-key`, `-k` | The path of your SSH identity key. Defaults to `~/.ssh/quay_installer` if left unspecified.
+| `--sslCert` | The path to the SSL/TLS certificate. Defaults to `{quayRoot}/quay-config` if left unspecified.
```

Comment:
```suggestion
| `--sslCert` | The path to the TLS public key / certificate. Defaults to `{quayRoot}/quay-config` and will be auto-generated if left unspecified.
```

Reply:
Required to be written as SSL/TLS. Fixed.